examples/server-user-manager.py

```diff
@@ -0,0 +1,43 @@
+import sys
+sys.path.insert(0, "..")
+import time
+
+try:
+    from IPython import embed
+except ImportError:
+    import code
+
+    def embed():
+        vars = globals()
+        vars.update(locals())
+        shell = code.InteractiveConsole(vars)
+        shell.interact()
+
+
+from opcua import ua, Server
+
+def simple_user_manager(isession, name, pwd):
```

Comment:
same issue as the other PR. It might not be your code, but when readong the word manager I am expecting a class with a few methods.

Reply:
This user_manager stuff loks very strange. Ideally it should have been an interface class

class UserManagerInterface:
     def check_user_token(self, isession, token):
            copy code
     def check_password(self, isession, user, passwd)
           raise NotImplementedError

then have a default one:
class SimpleUserManager(UserManagerInterface):
         def check_password(self, isession, user_name, passwd):
                 if self.allow_remote_admin and userName in ("admin", "Admin"):
                         isession.user = UserManager.User.Admin
                 return True

Reply:
> same issue as the other PR. It might not be your code, but when readong the word manager I am expecting a class with a few methods.

Fully agreed, this makes more sense. So let's drop the example code for now and add it once the user manager code stabilizes.

Reply:
@helmut-jacob do you look at this user manager stuff or should I do it? I am really busy currently so it might take a while